composer.json

```diff
@@ -10,7 +10,9 @@
         "ast"
     ],
     "require": {
-        "rector/rector-prefixed": "0.7.29"
+        "php": "^7.2",
```

Comment:
PHP version must be part of `composer.json`, so composer can work with direct minimal version of this package.

This is minimal requirement of Rector 7.x

Reply:
That makes sense, thanks.